packages/react-conformance/src/defaultTests.tsx

```diff
@@ -0,0 +1,69 @@
+import { TestObject, TestingOptions } from './types';
+import { ComponentDoc } from 'react-docgen-typescript';
+import { mount } from 'enzyme';
+import * as ReactIs from 'react-is';
+import * as React from 'react';
+
+/** For a reference of the tests, check the file rules.md in @fluentui/react-conformance */
+
+export const defaultTests: TestObject = {
+  /** Component has a docblock with 5 to 25 words */
+  'has-docblock': (componentInfo: ComponentDoc, componentPath: string, testInfo: TestingOptions) => {
+    const maxWords = 25;
+    const minWords = 5;
+
+    // No need to check if the description is undefined, ComponentDoc.description is a "string" not "string | undefined"
+    it(`has a docblock with ${minWords} to ${maxWords} words`, () => {
+      expect(componentInfo.description).toBeGreaterThanOrEqual(minWords);
+      expect(componentInfo.description).toBeLessThanOrEqual(maxWords);
+    });
+  },
+  /** Component file exports a valid React element type  */
+  'exports-react-element': (componentInfo: ComponentDoc, componentPath: string, testInfo: TestingOptions) => {
+    it(`file exports valid React Element type`, () => {
+      expect(ReactIs.isValidElementType(require(componentPath))).toBe(true);
+    });
+  },
+  /** Component file exports a valid React element and can render it */
+  'component-renders': (componentInfo: ComponentDoc, componentPath: string, testInfo: TestingOptions) => {
+    const { requiredProps } = testInfo;
+    const Component = require(componentPath);
```

Comment:
You should move this code within the `it()` so that it will be run within the context of the actual test (so failures are linked to the right place). Same for the setup of all the other tests.

Also, `require(componentPath)` will get you the file's entire exports object, not just the component. To get the component, which may be either a default or named export, you should do:
```
const componentFile = require(componentPath);
const Component = componentFile.default || componentFile[componentName];
expect(Component).toBeDefined();
```